freeipa.spec.in

```diff
@@ -617,6 +617,7 @@ If you are installing an IPA server, you need to install this package.
 Summary: IPA integrated DNS server with support for automatic DNSSEC signing
 BuildArch: noarch
 Requires: %{name}-server = %{version}-%{release}
+Requires: unbound
```

Comment:
I wonder if we should create a separate subpackage (freeipa-server-dns-encrypted) that should contain these dependencies instead. And require its presence in the installer code but only `Recommends:` in the `freeipa-server-dns-encrypted`.

This would allow us not to have unbound and friends to be always installed everywhere even if people don't want to use encrypted DNS. Make this requirement a condition for `--dns-policy=enforced` + any of `--dns-tls-*` options.

Also, since we are using unbound on the client, then `%package server-dns-encrypted` can simply have `Requires: freeipa-client-dns-encrypted` which contains unbound dependency.